src/library/librarycontrol.cpp

```diff
@@ -205,15 +218,15 @@ void LibraryControl::bindWidget(WLibrary* pLibraryWidget, KeyboardEventFilter* p
 }
 
 void LibraryControl::libraryWidgetDeleted() {
```

Comment:
huh, it's been a while since I looked at this code -- we could use a QPointer for these pointers which automatically gets cleared when the QObject pointed to is deleted. (just noting, no need in this pr)